doc/api/util.md

```diff
@@ -167,6 +167,14 @@ corresponding argument. Supported placeholders are:
 * `%f` - Floating point value.
 * `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument
 contains circular references.
+* `%o` - Object. A string representation of an object
+  with generic JavaScript object formatting.
+  Similar to `util.inspect()` with options `{ showHidden: true, depth: 4, showProxy: true }`.
```

Comment:
Even though I didn't originate it, IMHO it comes from `optimally useful formatting` and 2 is too small.
Since we can't do interactive expansion like in a GUI, Ideally I'd say it needs to be `Infinity`, but that's unreasonable.

This does make me think about a configurable `util.inspect.extendedFormattingOptions` or even `util.format.optimallyUsefulFormatter` but that could happen in a later PR.